deck-o-cards.js

```diff
@@ -1,43 +1,64 @@
+var prompt = require('./tools/prompt-sync')();
+var yn = require('./tools/yn');
+
 // 1. Make the function deck_o_cards assemble an array of cards using the provided suits and values arrays.
 // Each card in the deck should be an object formatted as: {suit: 'hearts', value: 'A'}
 function deck_o_cards() {
-  var values = [2, 3, 4, 5, 6, 7, 8, 9, 10, 'Jack', 'Queen', 'King', 'Ace'];
-  var suits = ['hearts', 'diamonds', 'clubs', 'spades'];
-
+  var values = ['A', 2, 3, 4, 5, 6, 7, 8, 9, 10, 'J', 'Q', 'K'];
+  var suits = ['♥', '♦', '♣', '♠'];
   var cards = []; // deck
   var shuffledCards = []; // deck shuffled
 
   // Make 52 card objects and store them in the "cards" array
-  // Hint: use 2 for loops
-
+  for (var s = 0; s < suits.length; s++) {
+    for(var v = 0; v < values.length; v++) {
+      cards.push({suit: suits[s], value: values[v]});
+    }
+  }
 
   // 2. Shuffle the cards
-  // Hint: shuffle function is already defined below
-
-
-  // Pull the top card from the newly shuffledCards
-
+  shuffledCards = shuffle(cards);
 
   // 3. Print the results:
-  // "The deck has {n} cards"
-  // "The top card is the {value} of {suit}"
+  console.log(`The Original Deck has ${cards.length} cards.`);
+  console.log(`The top card is ${cardToString(cards[0])}`);
+  printFullDeck(cards);
+  console.log("----");
+  console.log(`The Shuffled Deck has ${shuffledCards.length} cards.`);
+  console.log(`The top card is ${cardToString(shuffledCards[0])}`);
```

Comment:
With ES6 you can do line in the backticks and don't have to reprint `console.log` each time.